setup.py

```diff
@@ -44,6 +44,8 @@ def run_tests(self):
     ext_package='cmocean',
     scripts = [],
     keywords = ['colormaps', 'oceanography', 'plotting', 'visualization'],
+    setup_requires=['setuptools'],
+    install_requires=['matplotlib', 'numpy'],
```

Comment:
Is it worth doing the "read the requirements.txt file" trick here (to make sure these never get out of sync)?

Reply:
Depends on how often these dependencies will change. Let me know if you want me to do that.

Reply:
@kthyng do you have any opinions on this?

Reply:
Ack, sorry that I neglected this. I don't care too much but I do not expect the dependencies to change much if ever!